### PR TITLE
Fix exception

### PR DESCRIPTION
**Pull request by MathiasReker**
Added check my length of email

### CODE REVIEW

File: contactform.php [hunk 1]
```diff
@@ -441,6 +441,12 @@ public function sendMessage()
                 [],
                 'Shop.Notifications.Error'
             );
+        } elseif (strlen(Tools::getValue('from')) > 255) {
```
Comment (Quetzacoalt91) [inline contactform.php]: What happens for previous version of PrestaShop with a shorter field length? 
I guess it should take the value from Contact::$definition if available.

Comment (MathiasReker) [reply contactform.php]: If PS < 1.7.5 with this fix:

without this PR:
=255: error 500 
>255: error 500
<255: send email

with this PR:
=255: error 500 //what is the chance?
>255: display error message
<255: send email

If PS = 1.7.5 with this fix:
with this PR:
=255: display error message
>255: display error message
<255: send email

Comment (Progi1984) [reply contactform.php:444]: @MathiasReker I follow @Quetzacoalt91. Please use the definition of fhe field.

**Comment by PierreRambaud** [comment]
Since we had no news from you for more than 30 days, I'll close this pull request. Feel free to reopen or open another one if you think it's still relevant.

Thanks!

**Pull request closed without merging**